!Command
command:
- concatenate_domains.py
positional: []
named: []
parent:
subcommands: []
usage: []
help_flag:
usage_flag:
version_flag:
help_text: "2020-09-10 00:49:11,285 -WARNING- Grid computing is not available because\
  \ DRMAA not configured properly: Could not find drmaa library.  Please specify its\
  \ full path using the environment variable DRMAA_LIBRARY_PATH\n2020-09-10 00:49:11,285\
  \ -INFO- No DRMAA, Switching to local/cluster mode.\nTraceback (most recent call\
  \ last):\n  File \"/usr/local/bin/concatenate_domains.py\", line 81, in <module>\n\
  \    main(inSeq=inSeq, domains=domains, outSeq=sys.stdout)\n  File \"/usr/local/bin/concatenate_domains.py\"\
  , line 13, in main\n    for rc in SeqIO.parse(inSeq, 'fasta'):\n  File \"/usr/local/lib/python3.8/site-packages/Bio/SeqIO/FastaIO.py\"\
  , line 178, in FastaIterator\n    with as_handle(handle, \"rU\") as handle:\n  File\
  \ \"/usr/local/lib/python3.8/contextlib.py\", line 113, in __enter__\n    return\
  \ next(self.gen)\n  File \"/usr/local/lib/python3.8/site-packages/Bio/File.py\"\
  , line 120, in as_handle\n    with open(handleish, mode, **kwargs) as fp:\nFileNotFoundError:\
  \ [Errno 2] No such file or directory: '--help'\n"
generated_using:
- --help
